### PR TITLE
Em 22

### PR DESCRIPTION
**Pull request by lalodiazdiaz**

### CODE REVIEW

**Review by ErickIsra01 — DISMISSED** [review]

File: src/components/Home/Home.jsx [hunk 1]
```diff
@@ -1,74 +1,56 @@
 import React, { useCallback, useState } from 'react';
-import Modal from 'react-modal';
+import Modal from '../Modal/Modal';
 import Navbar from '../Navbar/Navbar';
 import styles from './Home.module.css';
 
-const CUSTOMSTYLES = {
-	content: {
+const url = ' https://wa.me/526145153963';
```
Comment (FraGa96) [inline src/components/Home/Home.jsx]: I suggest use this as an env var or a constant in an external file like /src/constants.js

File: src/components/Home/Home.jsx [hunk 2]
```diff
@@ -1,74 +1,56 @@
...
-		top: '50%',
-		transform: 'translate(-50%, -50%)',
-		width: '70%',
-	},
-};
+function contModal() {
+	return (
+		<div className={styles.contact}>
+			<h2>Lic. Claudia Patricia Gonzalez Moreno </h2>
+			<a href={url} target="_blanket">
```
Comment (FraGa96) [inline src/components/Home/Home.jsx]: did you mean "_blank"?

File: src/components/Home/Home.jsx [hunk 3]
```diff
@@ -1,74 +1,56 @@
...
+				<button
```
Comment (FraGa96) [inline src/components/Home/Home.jsx]: remove the button tag and style the anchor tag as button

File: src/components/Home/Home.jsx [hunk 4]
```diff
@@ -1,74 +1,56 @@
...
+					className={styles.llamarPsicologo}
+					type="button"
+				>Contactar
```
Comment (FraGa96) [inline src/components/Home/Home.jsx]:
```suggestion
				>
				  Contactar
```

File: src/components/Home/Home.jsx [hunk 5]
```diff
@@ -1,74 +1,56 @@
...
+				<div className={styles.bodyHome}>
 					<button
-						className={styles.btnCloseModal}
-						onClick={closeModal}
+						className={styles.agendar}
+						onClick={modalOpenAction}
 						type="button"
-					>
-						Cerrar
+					>Agendar
```
Comment (FraGa96) [inline src/components/Home/Home.jsx]:
```suggestion
					>
					  Agendar
```

File: src/components/Home/Home.module.css [hunk 6]
```diff
@@ -16,94 +22,123 @@
...
-  border-radius:50px;
+  border-radius:5px;
   width: 130px;
-  background: rgba(13, 177, 108, 0.79); 
-  border: 2px solid black;
+  background: #66d086;
+  border: 0;
 }
 
 .agendar:hover{
```
Comment (FraGa96) [inline src/components/Home/Home.module.css]: english please

File: src/components/Home/Home.module.css [hunk 7]
```diff
@@ -16,94 +22,123 @@
...
   justify-content: space-around;
   align-items: center;
-  margin: 20px;
+  
 }
-
-.inputCita{
-  color: black;
-  padding: 15px 32px;
+.llamarPsicologo{
```
Comment (FraGa96) [inline src/components/Home/Home.module.css]: english please

File: src/components/Home/Home.module.css [hunk 8]
```diff
@@ -16,94 +22,123 @@
...
+  background: #66d086;
+  border-color: #66d086;
 }
-
-.modalCont > text{
-  font-size: 20px;
-  margin-left: 10px;
-  color: black;
-  font-weight: 500;
+.llamarPsicologo:hover{
```
Comment (FraGa96) [inline src/components/Home/Home.module.css]: english please x3

File: src/components/Home/Home.module.css [hunk 9]
```diff
@@ -16,94 +22,123 @@
...
-  padding: 15px 32px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 16px;
-  background-color:  rgba(13, 177, 108, 0.79);
-  border: 2px solid black;
-  border-radius:50px;
-  width: 130px;
+  .agendar{
```
Comment (FraGa96) [inline src/components/Home/Home.module.css]: english please x4

Check the file and use only english

File: src/components/Home/Home.module.css [hunk 10]
```diff
@@ -16,94 +22,123 @@
...
+
+  .contact h2{
+    font-size: 30px;
+    text-align: center;
+  }
+  .llamarPsicologo{
+  height: max-content;
+  width: 150px;
+  font-size: 25px;
+  margin-right: 19px;
```
Comment (FraGa96) [inline src/components/Home/Home.module.css]: fix the indentation

File: src/components/Home/Home.module.css [hunk 11]
```diff
@@ -16,94 +22,123 @@
...
-    font-size: 30px;
-    padding: 2%;
-    height: 30px;
+  .contModal h1{
+    font-size: 25px;
   }
+
+  .contact{
+  flex-direction: column;
+  justify-content: center;
```
Comment (FraGa96) [inline src/components/Home/Home.module.css]: Fix the indentation

File: src/components/Home/Home.module.css [hunk 12]
```diff
@@ -16,94 +22,123 @@
...
+  }
+
+  .contact h2{
+  font-size: 20px;
+  text-align: center;
+  }
+.llamarPsicologo{
+  font-size: 20px;
+  width: 120px;
+}
```
Comment (FraGa96) [inline src/components/Home/Home.module.css]: fix the indentation

File: src/components/Home/Home.jsx [hunk 13]
```diff
@@ -1,74 +1,56 @@
...
 
-			</Modal>
-		</div>
+			</div>
+			<Modal
+				Content={contModal}
+				onAction={modalCloseAction}
+				style={modal}
+				title="Consulta a tu psicologo"
+			/>
```
Comment (FraGa96) [inline src/components/Home/Home.jsx]:
```suggestion
			<Modal
				Content={contModal}
				onAction={modalCloseAction}
				style={modal}
				title="Consulta a tu psicologo"
			>
			 {contModal}
			</Modal>
```

File: src/components/Home/Home.jsx [hunk 14]
```diff
@@ -1,74 +1,56 @@
...
-						Cerrar
+					>Agendar
 					</button>
 				</div>
 
-			</Modal>
-		</div>
+			</div>
+			<Modal
+				Content={contModal}
```
Comment (FraGa96) [inline src/components/Home/Home.jsx]: use children instead

File: src/components/Modal/Modal.jsx [hunk 15]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './Modal.module.css';
+
+function Modal({ style, onAction, title, Content }) {
```
Comment (FraGa96) [inline src/components/Modal/Modal.jsx]:
```suggestion
function Modal({ style, onAction, title, children}) {
```

File: src/components/Modal/Modal.jsx [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+	return (
+		<div className={style}>
+			<div className={styles.backModal} />
+			<div className={styles.contModal}>
+				<h1>{title}</h1>
+				<Content />
```
Comment (FraGa96) [inline src/components/Modal/Modal.jsx]:
```suggestion
				{children}
```

File: src/components/Modal/Modal.module.css [hunk 17]
```diff
@@ -0,0 +1,118 @@
...
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: column;
+  background-color: white;
+  height: max-content;
+  width: 45%;
+  border: 3px #b2b9ba solid;
+  border-radius: 5px;
+}
+.contact{
```
Comment (FraGa96) [inline src/components/Modal/Modal.module.css]: missing empty line, separate each class with an empty line
also leave a space between the open bracket and the class name (check this in the 3 .modules.css files)
```
.classname {
```

File: src/components/Modal/Modal.module.css [hunk 18]
```diff
@@ -0,0 +1,118 @@
...
+
+.contact h2{
+  font-size: 20px;
+  text-align: center;
+}
+.closeModal{
+  height: max-content;
+  width: 120px;
+  font-size: 20px;
+}
```
Comment (FraGa96) [inline src/components/Modal/Modal.module.css]: fix indentation

File: src/components/Navbar/Navbar.jsx [hunk 19]
```diff
@@ -7,7 +7,7 @@ function Navbar() {
 			<span className={styles.navLogo}>BEGINNING</span>
 			<div className={styles.navItems}>
 				<Link className={styles.link} to="login">
-					<div className={styles.btnLog}>LogIn</div>
+					<button className={styles.btnLog} type="button">LogIn</button>
```
Comment (FraGa96) [inline src/components/Navbar/Navbar.jsx]: button is not required if it is already a Link

**Review by FraGa96 — CHANGES_REQUESTED** [review]

File: src/constants.js [hunk 20]
```diff
@@ -0,0 +1,2 @@
+const URL = ' https://wa.me/526145153963';
+export default URL;
```
Comment (FraGa96) [inline src/constants.js:2]: this file is suposed to export multiple constants at one, don't use export default
use this instead
```
export const URL = ...
```

**Review by FraGa96 — CHANGES_REQUESTED** [review]

**Review by FraGa96 — APPROVED** [review]

**Review by ErickIsra01 — APPROVED** [review]

**Pull request merged**